pkg/drivers/kic/oci/network_create.go

```diff
@@ -134,11 +134,21 @@ type netInfo struct {
 	gateway net.IP
 	mtu     int
 }
+type networkInspect struct {
```

Comment:
could add a comment that this struct is only used for unmarshal the docker output and later it will be translated to netinfo